.github/workflows/main.yml

```diff
@@ -71,10 +71,10 @@ jobs:
         with:
           path: ~/.cache/go-build
           key: ${{ runner.os }}-gobuild-${{ hashFiles('**/*.go') }}
-      - name: Setup Go 1.15
+      - name: Setup Go 1.15.7
         uses: actions/setup-go@v1
         with:
-          go-version: 1.15
+          go-version: 1.15.7
```

Comment:
Adding the patch level here doesn't seem to add any advantage since `1.15` uses the latest patch anyway. It just means we have to update this every time there's a new version of Go available vs. automatically picking it up.

Reply:
Thanks for the feedback @nojnhuh - removed it!